public/employers/staples.yml

```diff
@@ -0,0 +1,34 @@
+name: Staples Inc.
+shortName: Staples
+wiki: Staples Inc.
```

Comment:
They have subsidiaries called Quill Corporation and Mondoffice, which is their brand in Italy.